website/content/en/docs/launch-templates.md

```diff
@@ -0,0 +1,207 @@
+---
+title: "Launch Templates and Custom Images"
+linkTitle: "Launch Templates"
+weight: 80
+---
+
+## Introduction
+
+Karpenter follows existing AWS patterns for customizing the base image of
+instances. More specifically, Karpenter uses [EC2 launch templates](https://docs.aws.amazon.com/autoscaling/ec2/userguide/LaunchTemplates.html). Launch
+templates may specify many values. The pivotal value is the base image (AMI).
+Launch templates further specify many parameters related to networking,
+authorization, instance type, and more.  
+
+This guide describes requirements for using launch templates with Karpenter, and includes an example procedure.
+
+## Launch Template Requirements
+
+### AMI
+
+AMI (Amazon Machine Image), is the base image/VM for a launch template.
+
+[Review the instructions for importing a VM to AWS.](https://docs.aws.amazon.com/vm-import/latest/userguide/vmimport-image-import.html) Note the AMI id generated by this process, such as, 
+`ami-074cce78125f09d61`.
+
+### User Data - Autoconfigure
+
+Importantly, the AMI must support automatically connecting to a cluster based
+on "user data", or a base64 encoded string passed to the instance at startup.
+The syntax and purpose of the user data varies between images. The Karpenter
+default OS, Amazon Linux 2 (AL2), accepts shell scripts (bash commands). 
+
+[AWS calls data passed to an instance at launch time "user
+data".](https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/user-data.html#user-data-shell-scripts)
+
+In the default configuration, Karpenter uses an EKS optimized version of AL2 and passes the hostname of the Kubernetes API server, and a certificate. The EKS Optimized AMI includes a `bootstrap.sh` script which connects the instance to the cluster, based on the passed data. 
+
+Alternatively, you may reference AWS's [`bootstrap.sh`
+file](https://github.com/awslabs/amazon-eks-ami/blob/master/files/bootstrap.sh)
+when building a custom base image. 
+
+```
+#!/bin/bash
+/etc/eks/bootstrap.sh <my-cluster-name> \
+--kubelet-extra-args <'--max-pods=40'> \

```

Comment:
Specifying `max-pods` can break Karpenter's binpacking logic (it has no way to know what this setting is). If karpenter attempts to pack more than this number of pods, the instance may be oversized, and additional pods will reschedule. Can we provide a disclaimer?